File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroRES0102U00GrdRES0103ResInfo.java
package nta.med.data.model.ihis.nuro;

import java.util.Date;

public class NuroRES0102U00GrdRES0103ResInfo {
	private String doctor;
    private Date jinryoPreDate;
    private String resAmStartHhmm;
    private String resAmEndHhmm;
    private String resPmStartHhmm;
    private String resPmEndHhmm;
    private String remark;
    private String amGwaRoom;
    private String pmGwaRoom;
	public NuroRES0102U00GrdRES0103ResInfo(String doctor, Date jinryoPreDate,
			String resAmStartHhmm, String resAmEndHhmm, String resPmStartHhmm,
			String resPmEndHhmm, String remark, String amGwaRoom,
			String pmGwaRoom) {
		super();
		this.doctor = doctor;
		this.jinryoPreDate = jinryoPreDate;
		this.resAmStartHhmm = resAmStartHhmm;
		this.resAmEndHhmm = resAmEndHhmm;
		this.resPmStartHhmm = resPmStartHhmm;
		this.resPmEndHhmm = resPmEndHhmm;
		this.remark = remark;
		this.amGwaRoom = amGwaRoom;
		this.pmGwaRoom = pmGwaRoom;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public Date getJinryoPreDate() {
		return jinryoPreDate;
	}
	public void setJinryoPreDate(Date jinryoPreDate) {
		this.jinryoPreDate = jinryoPreDate;
	}
	public String getResAmStartHhmm() {
		return resAmStartHhmm;
	}
	public void setResAmStartHhmm(String resAmStartHhmm) {
		this.resAmStartHhmm = resAmStartHhmm;
	}
	public String getResAmEndHhmm() {
		return resAmEndHhmm;
	}
	public void setResAmEndHhmm(String resAmEndHhmm) {
		this.resAmEndHhmm = resAmEndHhmm;
	}
	public String getResPmStartHhmm() {
		return resPmStartHhmm;
	}
	public void setResPmStartHhmm(String resPmStartHhmm) {
		this.resPmStartHhmm = resPmStartHhmm;
	}
	public String getResPmEndHhmm() {
		return resPmEndHhmm;
	}
	public void setResPmEndHhmm(String resPmEndHhmm) {
		this.resPmEndHhmm = resPmEndHhmm;
	}
	public String getRemark() {
		return remark;
	}
	public void setRemark(String remark) {
		this.remark = remark;
	}
	public String getAmGwaRoom() {
		return amGwaRoom;
	}
	public void setAmGwaRoom(String amGwaRoom) {
		this.amGwaRoom = amGwaRoom;
	}
	public String getPmGwaRoom() {
		return pmGwaRoom;
	}
	public void setPmGwaRoom(String pmGwaRoom) {
		this.pmGwaRoom = pmGwaRoom;
	}
    
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroRES0102U00GrdRES0103ResInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroRES0102U00GrdRES0103ResInfo
    {
        private String _doctor;
        private String _jinryoPreDate;
        private String _resAmStartHhmm;
        private String _resAmEndHhmm;
        private String _resPmStartHhmm;
        private String _resPmEndHhmm;
        private String _remark;
        private String _amGwaRoom;
        private String _pmGwaRoom;
        private String _dataRowState;

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String JinryoPreDate
        {
            get { return this._jinryoPreDate; }
            set { this._jinryoPreDate = value; }
        }

        public String ResAmStartHhmm
        {
            get { return this._resAmStartHhmm; }
            set { this._resAmStartHhmm = value; }
        }

        public String ResAmEndHhmm
        {
            get { return this._resAmEndHhmm; }
            set { this._resAmEndHhmm = value; }
        }

        public String ResPmStartHhmm
        {
            get { return this._resPmStartHhmm; }
            set { this._resPmStartHhmm = value; }
        }

        public String ResPmEndHhmm
        {
            get { return this._resPmEndHhmm; }
            set { this._resPmEndHhmm = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String AmGwaRoom
        {
            get { return this._amGwaRoom; }
            set { this._amGwaRoom = value; }
        }

        public String PmGwaRoom
        {
            get { return this._pmGwaRoom; }
            set { this._pmGwaRoom = value; }
        }

        public String DataRowState
        {
            get { return this._dataRowState; }
            set { this._dataRowState = value; }
        }

        public NuroRES0102U00GrdRES0103ResInfo() { }

        public NuroRES0102U00GrdRES0103ResInfo(String doctor, String jinryoPreDate, String resAmStartHhmm, String resAmEndHhmm, String resPmStartHhmm, String resPmEndHhmm, String remark, String amGwaRoom, String pmGwaRoom, String dataRowState)
        {
            this._doctor = doctor;
            this._jinryoPreDate = jinryoPreDate;
            this._resAmStartHhmm = resAmStartHhmm;
            this._resAmEndHhmm = resAmEndHhmm;
            this._resPmStartHhmm = resPmStartHhmm;
            this._resPmEndHhmm = resPmEndHhmm;
            this._remark = remark;
            this._amGwaRoom = amGwaRoom;
            this._pmGwaRoom = pmGwaRoom;
            this._dataRowState = dataRowState;
        }

    }
}